runtime/doc/options.txt

```diff
@@ -5571,6 +5571,15 @@ A jump table for the options with a short description can be found at |Q_op|.
 	NOTE: This option is set to the Vi default value when 'compatible' is
 	set and to the Vim default value when 'compatible' is reset.
 
+                                                *'relativenumberformat'* *'rnfm'*
+'relativenumberformat' 'rnfm'	string	(default "%-*ld ")
+                        global
```

Comment:
The 'relativenumber' option is "local to window". Should't be the same?

Reply:
Make sense.